.gitignore

```diff
@@ -4,6 +4,7 @@
 bin/*
 log/*
 .ipynb_checkpoints/
+.idea
```

Comment:
We don't want IDE specific entries in the .gitignore file as every new contributor could potentially add another entry. Please set up a global gitignore on your machine for those.